docs/tech-reference/iso-rsync-distributor.rst

```diff
@@ -0,0 +1,70 @@
+=====================
+ISO rsync Distributor
+=====================
+
+Purpose:
+========
+The ISO rsync distributor publishes ISO content to a remote server. The distributor uses rsync over
+ssh to perform the file transfer.
+
+Configuration
+=============
+Here's an example of rpm_rsync_distributor configuration:
+
+.. code-block:: json
+
+    {
+     "distributor_id": "my_iso_rsync_distributor",
+     "distributor_type_id": "iso_rsync_distributor",
+     "distributor_config": {
+        "remote": {
+            "auth_type": "publickey",
+            "ssh_user": "foo",
+            "ssh_identity_file": "/home/user/.ssh/id_rsa",
+            "host": "192.168.121.1",
+            "root": "/home/foo/pulp_root_dir"
+        },
+        "predistributor_id": "my_iso_distributor",
+        }
+    }
+
+
+``predistributor_id``
+  The id of the iso_distributor associated with the same repository. The PULP_MANIFEST published by
+  the predistributor is copied to the remote server.
+
+The ``distributor_config`` contains a ``remote`` section made up of the following settings:
+
+``auth_type``
+  Two authentication methods are supported: ``publickey`` and ``password``.
+
+``ssh_user``
+  ssh user for remote server
+
+``ssh_identity_file``
+  A path to the private key that will be used as identity file for ssh. When ``auth_type`` is
+  ``publickey`` this is a required config. The key has to be readable by user ``apache``.
+
+``ssh_password``
+  Password for ``ssh_user`` on remote server. Password is required when ``auth_type`` is 'password'.
```

Comment:
s/Password for /The password to be used for /
s/on remote/on the remote/
s/Password is/``ssh_password`` is/